src/patternfly/components/Table/table.scss

```
@@ -787,9 +787,11 @@
 
   &.pf-m-favorited .#{$button} {
     --#{$button}--m-plain--Color: var(--#{$table}__favorite--m-favorited--c-button--Color);
+    --#{$button}--m-plain__icon--Color: var(--#{$table}__favorite--m-favorited--c-button--Color);
 
     &:is(:hover, :focus) {
       --#{$button}--m-plain--hover--Color: var(--#{$table}__favorite--m-favorited--c-button--Color);
+      --#{$button}--hover__icon--Color: var(--#{$table}__favorite--m-favorited--c-button--Color);
```

Nice!! You should be able to just replace the existing vars with yours - can you test this and see if it works as expected? The plain icon in this case only has an icon in it, so we should only need to set the `__icon` vars.

```suggestion
    --#{$button}--m-plain__icon--Color: var(--#{$table}__favorite--m-favorited--c-button--Color);

    &:is(:hover, :focus) {
      --#{$button}--hover__icon--Color: var(--#{$table}__favorite--m-favorited--c-button--Color);
```

---

Yes, works fine even when replaced